client/components/horizontal-list/index.js

```diff
@@ -3,16 +3,37 @@
 /**
  * External dependencies
  */
-import { List } from '@woocommerce/components';
+import * as React from 'react';
+import { ExperimentalList } from '@woocommerce/experimental/build/experimental-list/experimental-list';
+import { ExperimentalListItem } from '@woocommerce/experimental/build/experimental-list/experimental-list-item';
 
 /**
  * Internal dependencies.
  */
 import './style.scss';
 
 const HorizontalList = ( props ) => {
-	const { items } = props;
-	return <List className="woocommerce-list--horizontal" items={ items } />;
+	const items = props.items.map( ( item ) => {
+		return (
+			<ExperimentalListItem key={ item.title }>
+				<div className="woocommerce-experimental-list__item-inner">
+					<span className="woocommerce-experimental-list__item-title">
+						{ item.title }
+					</span>
+					<span className="woocommerce-experimental-list__item-content">
+						{ item.content }
+					</span>
+				</div>
+			</ExperimentalListItem>
+		);
+	} );
+
+	return (
+		<ExperimentalList
+			className="woocommerce-experimental-list--horizontal wcpayments-horizontal-list"
+			children={ items }
+		/>
+	);
```

Comment:
I'm not sure how to test this. 🤔

This doesn't seem to be part of the scope of this PR. Is it?

Reply:
Somehow I ended up merging another branch inside of the one I was working. I've reverted the merge now